### ISSUE TITLE
Updates for latest release

### ISSUE THREAD

**Issue by stevenhurth** [ionic-team/ionic-vue-conference-app#1]
Not really an issue, but more of an inquiry.

I was just curious if there are intentions on updating the ionic-vue-conference-app to work with the latest and greatest?

**Comment by treetopdevs** [comment]
I started down the road of upgrading all the npm dependancies, and got pretty far.

I got stuck in the types for VueX, and I'm not yet proficient enough with Typescript to unravel this easily.

This is the type of error I'm getting:
`ERROR in /ionic-vue-conference-app-master/src/store/index.ts(19,5):
19:5 Type 'Module<LocationState, {}>' is not assignable to type 'Module<any, StoreState>'.
  Type '{}' is missing the following properties from type 'StoreState': locations, sessions, speakers, user
    17 | export default new Vuex.Store<StoreState>({
    18 |   modules: {
  > 19 |     locations,
       |     ^
    20 |     sessions,
    21 |     speakers,
    22 |     user`

**Comment by jackflyer** [comment]
Yes, this should be updated to latest and greatest. @treetopdevs can you share repository , you got up to so far.

**Comment by laurensiusadi** [comment]
Yes, please update to latest, as of writing it's 5.6.10.
